website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+
+Will a genetically modified individual be legally [scope of legal definition required] denied citizenship or human rights by 20XX?
+
+Will more than XX% of countries adopt ethical guidelines [scope of ethical guidelines required] prohibiting the selection of embryos based on non-health-related traits by 20XX?
+
+Will human genome editing be used as a defense in a court of law in a high-profile case [defense in country and publicity of case definitions required] by 20XX?
+
+Will there be widespread protests / riots [scope of civil disobience surrounding topic required] against the use of polygenic screening in schools or workplaces [OR other settings] by 20XX?
+
+Will any country [some scope correction probably useful here] allow parents to choose the sexual orientation of their children via genetic screening or editing by 20XX?
+
+Will more than XX% of the global population use genetic enhancements [measurement for global population and use of genetic enhancement required] that impact physical appearance by 20XX?
+
+In what year will over XX% of countries have a regulatory framework for genetic editing of embryos [current log of countries with genetic editing frameworks required]?
+
+What percentage of the global [OR country OR country consortium] population will support [definition of support required] the right to use genetic engineering for enhancement by 20XX?
+
+In what year will the first international legal case [more detail on international legal case required] be filed concerning genetic discrimination in employment [OR other area]?
+
+By what year will more than XX% of the world’s population [OR country OR country consortium] have undergone some form of genetic enhancement for non-medical purposes [measurement for this question required]?
+
+In what year will more than XX% of healthcare systems globally [scope of global healthcare systems required] implement genetic screening as a routine part [definition of routine part required] of preventive care?
+
+By what year will more than 30 countries [scope of countries required] have established policies [scope of policies required] to protect against genetic inequality [definition of access required] in access to enhancements?
+
+What will be the average cost of a genetic enhancement procedure [stratification by procedure required] in high-income countries [scope and definition of high-income nations required, equivalent question for "low-income" nations required] by 20XX?
+
+What percentage of parents in OECD countries will choose to genetically screen their children for cognitive traits [OR other traits] by 20XX?
+
+By what year will more than XX% of the world's population support laws [definition of support laws required] that prevent genetic discrimination in employment and insurance [OR other areas]?
+
+In what year will a global ethical consensus [scope and definition of consensus requirement] on the limits of genetic editing for non-medical enhancements be reached?
+
```

Comment:
thought: "non-medical" is very likely to become a moving target in the future, as the scope of medical validity will be expanded to cover an increasingly broad range of traits (and medical+aesthetic pleiotropies will be selectively popularized);
probably this question should be made contingent on some firm line having been drawn between "medical" and "non-medical" enhancements, else it'll fail to clearly resolve